src/Invoker.php

```diff
@@ -44,6 +46,12 @@ public function __construct(callable $target, string $signatureType)
             throw new InvalidArgumentException(sprintf(
                 'Invalid signature type: "%s"', $signatureType));
         }
+        $this->errorLogFunc = function (string $error) {
+            fwrite(fopen('php://stderr', 'wb'), json_encode([
```

Comment:
Will this error also get routed to Error Reporting on Cloud Functions? (Sending further context in chat.)

Reply:
I hope so!

Reply:
We need to manually test. If it doesn't work, we might not want to make the change.